File: Problem_Sets/pset6/sentimental/mario/more/mario.py
from cs50 import get_int

def main():
    while True:
        hashNumber = get_int("Height: ");
        if (1 <= hashNumber <= 8):
            break

    for i in range(hashNumber):
        print(" " * (hashNumber-(i+1)), end="")
        print("#" * (i+1), end="")
        print("  ", end="")
        print("#" * (i+1))

if __name__ == "__main__":
    main()

File: Problem_Sets/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void pyramids(int n);
int get_right_int(string prompt);

int main(void)
{
    
    int hash_number = get_right_int("Pyramids Height? ");
    pyramids(hash_number);
}

int get_right_int(string prompt)
{
    int n;
    do 
    {
        n = get_int("%s", prompt);
    }
    while (n < 1 || n > 8);
    return n;
}

void pyramids(int n)
{
    int count=1;
    for (int i = n; i > 0; i--)
        {
            for (int k = i; k > 1; k--)
            {
                printf(" ");
            }
        
            for (int k = 0; k < count; k++)
            {
                printf("#");
            }
            printf("  ");
            for (int k = 0; k < count; k++)
            {
                printf("#");
            }
            printf("\n");
        
            count += 1;
        }
}
